go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubernetes-csi/external-snapshotter/v8
 
-go 1.23.1
+go 1.22.9
```

Comment:
Kubernetes already moved to 1.23.3: https://github.com/kubernetes/kubernetes/blob/master/.go-version
We can't move back.